dbhydra/dbhydra_core.py

```diff
@@ -1012,12 +1015,43 @@ def init_all_columns(cls, db1, name):
         return (cls(db1, name, columns, types))
 
     def get_all_columns_and_types(self):
-        results = self.db.client.get_table(self.name)
+        results = self.db1.client.get_table(f"{self.db1.credentials.project_id}.{self.db1.dataset}.{self.name}")
         column_names = [x.name for x in results.schema ]
         column_types = [x.field_type for x in results.schema]
         return column_names, column_types
 
 
+    def select(self, query):
+
+        """given SELECT query returns Python list"""
+        """Columns give the number of selected columns"""
+        print(query)
+        rows =  self.db1.client.query(query).result()
+        return rows
+        # column_string = query.lower().split("from")[0]
+        # if "*" in column_string:
+        #     columns = len(self.columns)
+        # elif column_string.find(",") == -1:
+        #     columns = 1
+        # else:
+        #     columns = len(column_string.split(","))
+        # rows = self.db1.cursor.fetchall()
+        # print(rows)
+        # if columns == 1:
+        #     cleared_rows_list = [item[0] for item in rows]
+        #
+        # if columns > 1:
+        #     cleared_rows_list = []
+        #     for row in rows:  # Because of unhashable type: 'pyodbc.Row'
+        #         list1 = []
+        #
+        #         for i in range(columns):
+        #             # print(row)
+        #             list1.append(row[i])
+        #         cleared_rows_list.append(list1)
+        # return (cleared_rows_list)
```

Comment:
Will this code be used again in the future? If so, leave a comment co everyone knows, else delete it.